app/decorators/course_decorator.rb

```diff
@@ -303,7 +303,9 @@ def not_on_find
   end
 
   def edit_vacancy_link
-    h.link_to("Edit", h.vacancies_provider_recruitment_cycle_course_path(provider_code: object.provider_code, recruitment_cycle_year: object.recruitment_cycle_year, code: object.course_code), class: "govuk-link")
+    h.link_to(h.vacancies_provider_recruitment_cycle_course_path(provider_code: object.provider_code, recruitment_cycle_year: object.recruitment_cycle_year, code: object.course_code), class: "govuk-link") do
+      h.raw("Edit<span class=\"govuk-visually-hidden\"> vacancies for #{name_and_code}</span>")
```

Comment:
This looks good here in the code, but in the screenshot on the PR there's no space before `vacancies`... 🤔 

Is this a similar strange bug to https://trello.com/c/hPSHwHbI/1491-change-links-check-your-answers-before-confirming ? Or an out of date screenshot?

Reply:
![you saw nothing](https://user-images.githubusercontent.com/470137/116242538-e9b9d800-a75d-11eb-828e-8c4f01bb7eef.gif)

devtool quirks

https://bugzilla.mozilla.org/show_bug.cgi?id=1304685